pkg/sql/schemachanger/scbuild/build.go

```diff
@@ -34,16 +34,16 @@ import (
 // The function takes an AST for a DDL statement and constructs targets
 // which represent schema changes to be performed.
 func Build(
```

Comment:
maybe take this opportunity to talk about `incumbent`?

Reply:
Done